orbit/pkg/table/authdb/authdb_darwin.go

```diff
@@ -0,0 +1,68 @@
+//go:build darwin
+// +build darwin
+
+package authdb
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os/exec"
+
+	"github.com/osquery/osquery-go/plugin/table"
+	"howett.net/plist"
+)
+
+// Columns is the schema of the table.
+func Columns() []table.ColumnDefinition {
+	return []table.ColumnDefinition{
+		table.TextColumn("right_name"), // required
+		table.TextColumn("json_result"),
+	}
+}
+
+// Generate is called to return the results for the table at query time.
+// Constraints for generating can be retrieved from the queryContext.
+func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	rightName := ""
+	if constraints, ok := queryContext.Constraints["right_name"]; ok {
+		for _, constraint := range constraints.Constraints {
+			if constraint.Operator == table.OperatorEquals {
+				rightName = constraint.Expression
+			}
+		}
+	}
+	if rightName == "" {
+		return nil, errors.New("missing right_name")
+	}
+
+	cmd := exec.Command("/usr/bin/security", "authorizationdb", "read", rightName)
+	out, err := cmd.Output()
+	if err != nil {
+		return nil, fmt.Errorf("generate failed: %w", err)
+	}
+
+	result, err := parseAuthDBReadOutput(out)
```

Comment:
The code looks very good.
(Future) comment for the sake of more unit-tests. Since we can't test `generate` (very hard to test it), I would make a function like 
`
convertAuthDBToJSON {
    parseAuthDBReadOutput(out)
    json.Marshal(result)
`
So the whole thing could be tested in addition to your `parseAuthDBReadOutput`.

Reply:
Yes, could be, reason I'm not doing so is that we would be testing the `json.Marshal` method can marshal a `map[string]interface{}` which is a test that should be on the golang standard library side. (Unless am missing some scenario that we'd like to test.)